test/patch/parse.js

```diff
@@ -248,8 +248,8 @@ Index: test2
     });
     it('should throw on invalid input in strict mode', function() {
       expect(function() {
-        parsePatch('Index: foo\nfoo', {strict: true});
-      }).to['throw'](/Unknown line 2 "foo"/);
+        parsePatch('Index: foo\n+++ bar\nblah', {strict: true});
+      }).to['throw'](/Unknown line 3 "blah"/);
```

Comment:
When I said positive case, I meant create a new test case that would be failing without the changes above. Basically I'd like to see a TDD artifact here so we both have a example of what this change fixes if someone looks back on this in the future. This also help protect yourself as it ensures that your use case doesn't regress in the future.